turbulence.gemspec

```diff
@@ -22,5 +22,5 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
-  s.required_ruby_version = '>= 1.8.7'
+  s.required_ruby_version = ">= 3.2.0"
```

Comment:
I've made a big jump here but only because this is a recent version of ruby. I did _not_ verify what specific version was necessary but I think it would make sense to switch to Prism for the parser.
I'm happy to walk back any of these changes where there might be concern that they were too aggressive in jumping versions.